nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-actions.js

```diff
@@ -1214,6 +1217,23 @@
         },
 
         /**
+         * Views the state for the specified processor.
+         *
+         * @param {selection} selection
+         */
+        changeVersion: function (selection) {
+            if (selection.size() !== 1 || !nf.CanvasUtils.isProcessor(selection)) {
+                return;
+            }
+
+            // get the processor data
+            var processor = selection.datum();
+
+            // attempt to change the version of the specified component
+            nf.ComponentVersion.promptForVersionChange(processor);
```

Comment:
Yup. Must have missed this one. Much of this was written prior to getting the modularization effort in. I went back through and updated each invocation using `nf.`. I'll update this one and make sure I didn't miss any others.